src/py42/exceptions.py

```diff
@@ -54,7 +54,12 @@ class Py42HTTPError(Py42ResponseError):
     """A base custom class to manage all HTTP errors raised by an API endpoint."""
 
     def __init__(self, exception, message=None):
-        message = message or u"Failure in HTTP call {}".format(str(exception))
+        error_message = ""
+        if hasattr(exception.response, "text"):
```

Comment:
`hasattr` is py3 only

Reply:
when would `exception.response` _not_ have a `text` attribute?

Reply:
some unit tests are written without passing response and those were affected by this change.